modules/tensor_mechanics/src/actions/CohesiveZoneActionBase.C

```diff
@@ -0,0 +1,152 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "CohesiveZoneActionBase.h"
+#include "CommonCohesiveZoneAction.h"
+#include "ActionWarehouse.h"
+#include "AddAuxVariableAction.h"
+#include "MooseEnum.h"
+
+// map vector name shortcuts to tensor material property names
+const std::map<std::string, std::string>
+    CohesiveZoneActionBase::_real_vector_cartesian_component_table = {
+        {"traction", "traction_global"},
+        {"jump", "displacement_jump_global"},
+        {"interface_jump", "interface_displacement_jump"},
+        {"pk1_traction", "PK1traction"}};
```

Comment:
Can we provide the meaning of those output names in the Action Mark Down file?

Reply:
done